modules/mysql/variables.tf

```diff
@@ -30,6 +30,12 @@ variable "random_instance_name" {
   default     = false
 }
 
+variable "database_version_replica" {
```

Comment:
This variable should be part of the [`read_replicas`](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/fa0d938a582febe6e4c3ce4cdc6aaa0b7a3b221e/modules/mysql/variables.tf#L181) slice since each replica can be upgraded individually